package.json

```diff
@@ -8,8 +8,17 @@
     "dist"
   ],
   "scripts": {
+    "prepublishOnly": "npm run build",
     "build": "node ./scripts/bundle.js && rollup -c",
-    "prettier": "prettier --write ."
+    "prettier-check-all": "prettier --check .",
+    "prettier-format-all": "prettier --write .",
+    "test": "npm run prettier-check-all"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm test",
+      "pre-push": "npm test"
```

Comment:
`pre-commit` should be enough, I think. Any reason to do it in `pre-push` as well?

Reply:
Husky uses both in their README https://github.com/typicode/husky#install so I'm not sure whether `pre-push` is needed.

Reply:
I removed `pre-push` hook.